ProcessLib/ComponentTransport/ComponentTransportProcessData.h

```diff
@@ -53,9 +46,7 @@ struct ComponentTransportProcessData
     {
     }
 
-    MaterialLib::PorousMedium::PorousMediaProperties porous_media_properties;
     ParameterLib::Parameter<double> const& fluid_reference_density;
```

Comment:
The fluid reference density seems to be also unused.

Reply:
I think so too. Shall I ?

Reply:
Removed.